design/role.go

```diff
@@ -70,6 +71,21 @@ var _ = a.Resource("resource_roles", func() {
 		a.Response(d.NotFound, JSONAPIErrors)
 		a.Response(d.Unauthorized, JSONAPIErrors)
 	})
+	a.Action("assignRole", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.PATCH("/:resourceID/roles/:roleName"),
+		)
+		a.Payload(updateUserIDList) // should refactor this variable's name in collaborators design definition too.
+		a.Description("Assigns an identity to a role, for a specific resource")
```

Comment:
Assigns a role to an identity :)

Reply:
a6581af88ea9d06a6d865829b2d6e03469942bb1 :D